PythonRegression/tests/features/steps/stitching_steps.py

```diff
@@ -0,0 +1,82 @@
+from aloe import step
+from util.test_logic import api_test_logic as tests
+from util.transaction_utils import transaction_logic as transactions
+
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+config = {}
+responses = {}
+
+side_tangle_address = b'SIDE9TANGLE9999999999999999999999999999999999999999999999999999999999999999999999'
+stitching_addresss = b'STITCHING9TRANSACTIONS99999999999999999999999999999999999999999999999999999999999'
+referencing_address = b'REFERENCES9STITCHING9TRANSACTION9999999999999999999999999999999999999999999999999'
+
+@step(r'a stitching transaction is issued on "([^"]*)" with the tag "([^"]*)"')
+def issue_stitching_transaction(step,node,tag):
+    config['nodeId'] = node
+    config['stitchTag'] = tag
```

Comment:
didn't we say we pass information between steps via world and a hook that cleans it between features/scenarios?

Reply:
We did talk about that yes, but since that will require me going through the other files and changing them as well, I thought it was better to split that into a different pr that focuses on just that.